.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Checklist
+
+- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
+- [ ] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
+- [ ] The description lists all applicable issues this PR seeks to resolve
+- [ ] The description lists any configuration setting(s) that differ from the default settings
+- [ ] All tests and CI builds passing
+- [ ] The description lists all relevant PRs included in this release _(remove this if not merging to master)_
+- [ ] Code coverage improves or is at 100% _(remove this if not merging to master)_
```

Comment:
Can you update the code coverage comment to match https://github.com/conveyal/datatools-ui/pull/411/commits/f4f1414079b2d86839988d59a1cc8c20781a4ad1?